### DISCUSSION TITLE
Edge Function 502 bad gateway

### DISCUSSION THREAD

**Discussion by justinfriebel** [vercel/community#3493] [Help]
### Summary

I have a SvelteKit Turso application that uses edge functions. The signin works locally but I'm getting this gateway error in the Vercel deploy.
```
Error: An internal error occurred with Vercel.

Your connection is working correctly.

Vercel is NOT working correctly.

This is an error with the platform itself, not the application.

502: BAD_GATEWAY
Code: BAD_GATEWAY
ID: cle1::kwx8h-1691086622515-e209117623a1
```

### Example

https://sveltekit-turso-drizzle-cdpqojaet-freebly.vercel.app/signin

### Steps to Reproduce

It's a template I've been working on so the code is open to look at: [https://github.com/justinfriebel/sveltekit-turso-drizzle/pull/4](https://github.com/justinfriebel/sveltekit-turso-drizzle/pull/4).

I've checked with Turso and they checked on the database and everything looked ok.

**Comment by aakash14goplani** [comment]
@justinfriebel - were you able to figure out solution for this problem?

**Reply by justinfriebel** [reply]
Unfortunately no. I thought someone from Vercel might have an idea but no response. It may be worth it to try a different host with the same code.
